src/Bicep.Core.UnitTests/Assertions/RootConfigurationAssertions.cs

```diff
@@ -24,11 +26,9 @@ public RootConfigurationAssertions(RootConfiguration configuration)
 
         public AndConstraint<RootConfigurationAssertions> HaveContents(string contents, string because = "", params object[] becauseArgs)
         {
-            string actual = StringUtils.ReplaceNewlines(Subject.ToUtf8Json(), "\n");
-            string expected = StringUtils.ReplaceNewlines(contents, "\n");
-
-            actual.Should().Be(expected, because, becauseArgs);
-
+            var actual = JToken.Parse(Subject.ToUtf8Json());
+            var expected = JToken.Parse(contents);
+            actual.Should().DeepEqual(expected, because, becauseArgs);
```

Comment:
@asilverman FYI we borrowed this from your branch - thanks for improving this assertion 😄